openshift/template.yaml

```diff
@@ -373,3 +428,70 @@ objects:
     selector:
       matchLabels:
         app: assisted-service
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: assisted-service-envoy-config
+data:
+  main.yaml: |
+    admin:
+      access_log_path: /dev/null
+      address:
+        socket_address: { address: 0.0.0.0, port_value: 9000 }
+
+    static_resources:
+      listeners:
+      - name: ingress
+        address:
+          socket_address: { address: 0.0.0.0, port_value: ${{ENVOY_PORT}} }
+        filter_chains:
+        - filters:
+          - name: envoy.filters.network.http_connection_manager
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+              access_log:
+              - name: envoy.access_loggers.file
+                typed_config:
+                  "@type": type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+                  path: /dev/stdout
+                  jsonFormat:
```

Comment:
I took a look in integration env, and I'd like to check something
Is there a way to order fields in this JSON by alphabetical order? It can come in handy when reading the output directly, even if we plan to let other tools read this stuff

Reply:
I am not aware of any option to force envoy to output ordered logs. I was also confused by the seemingly random order for log fields

Reply:
then no worries about it. it's just a micro-optimization